pkg/test/framework/analyzer.go

```diff
@@ -73,6 +74,11 @@ func (s *suiteAnalyzer) Label(labels ...label.Instance) Suite {
 	return s
 }
 
+func (s *suiteAnalyzer) SkipLabel(labels ...label.Instance) Suite {
```

Comment:
This is necessary because at a suite level we only skip for explicit "excludes" or "-label" in the selector to allow individual tests to be picked up even if the parent suite doesn't have the label (some examples with PostSubmit). 

This lets us exclude the pilot/ suite from the multicluster job.

Reply:
I understand the goal of this, but I am a bit confused why we want to skip multicluster job when this PR is about adding multicluster support?

Reply:
At least for now I'm ditching this in favor of https://github.com/istio/test-infra/pull/2840